backend/app/modules/facts_check/web_search.py

```
@@ -28,15 +28,24 @@
-    )
-    res = results.json()
-    first = {}
-    first["title"] = res["items"][0]["title"]
-    first["link"] = res["items"][0]["link"]
-    first["snippet"] = res["items"][0]["snippet"]
-
-    return [
-        first,
-    ]
+    try:
+        results = requests.get(
```

Ruff (0.15.6)

[error] Probable use of `requests` call without timeout

(S113)